espnet2/train/preprocessor.py

```diff
@@ -198,7 +200,7 @@ def __init__(
                         self.noises.append(sps[1])
             sps = noise_db_range.split("_")
             if len(sps) == 1:
-                self.noise_db_low, self.noise_db_high = float(sps[0])
```

Comment:
looks like that the previous `self.noise_db_low, self.noise_db_high = float(sps[0])` is a bug?

Reply:
Indeed. I believe this is a bug that was not found.